Revit_Core_Engine/Convert/Revit/ToRevit/Element.cs

```diff
@@ -214,9 +214,21 @@ private static Element ToRevitElement(this ModelInstance modelInstance, MEPCurve
         private static Element ToRevitElement(this ModelInstance modelInstance, FamilySymbol familySymbol, RevitSettings settings)
         {
             Document doc = familySymbol.Document;
-            if (modelInstance.Location is BH.oM.Geometry.Point)
+            if (AdaptiveComponentInstanceUtils.IsAdaptiveFamilySymbol(familySymbol))
+            {
+                List<IGeometry> pts = (modelInstance.Location as CompositeGeometry)?.Elements;
+                if (pts == null || !pts.All(x => x is BH.oM.Geometry.Point))
+                {
+                    BH.Engine.Reflection.Compute.RecordError($"A family could not be created based on the given ModelInstance because its family type is adaptive, but location was neither a collection of points. BHoM_Guid: {modelInstance.BHoM_Guid}");
```

Comment:
```suggestion
                    BH.Engine.Reflection.Compute.RecordError($"A family could not be created based on the given ModelInstance because its family type is adaptive, but location was not a collection of points. BHoM_Guid: {modelInstance.BHoM_Guid}");
```
Typo